### PR TITLE
Aflores 293563 peer review

### PR DESCRIPTION
**Pull request by afloresOSI**
Jason, the same edits apply to ADH.

### CODE REVIEW

**Review by jasnames — APPROVED** [review]
@afloresOSI Your changes all look good. Please review my updates to omf-editor and identity-providers. I published the updated branch to [dev](https://osisoft-dev.zoominsoftware.io/bundle/ocs-aflores-293563-peer-review/page/set-up/identity-providers.html).

File: content/developer-guide/omf-dev/omf-editor.md [hunk 1]
```diff
@@ -4,19 +4,26 @@ uid: omfEditor
...
+1. Select **Developer Tools** > **OMF Editor** from the left pane. 
 
-- Select **Copy OMF** to copy the message content to the clipboard. 
+1. Choose the message type from the **OMF Type** selector: **Type**, **Container** or **Data**.
 
-- Select **More Options** ![More Options](..\images\more-options.png "More Options") > **Format OMF** to reset template content alignment after making changes. 
+1. Select the OMF version from the **OMF Version** selector.
 
-- Select **More Options** ![More Options](..\images\more-options.png "More Options") > **Download OMF** to download the message content to your local drive. 
+2. Edit your message and use the following options:
```
Comment (afloresOSI) [inline content/developer-guide/omf-dev/omf-editor.md:19]:
```suggestion
1. Edit your message and use the following options:
```

File: content/developer-guide/omf-dev/omf-editor.md [hunk 2]
```diff
@@ -4,19 +4,26 @@ uid: omfEditor
...
-- Select **More Options** ![More Options](..\images\more-options.png "More Options") > **Reset Template** to delete any changes you have made to the message content. 
+   - Select **Copy OMF** to copy the message content to the clipboard. 
 
+   - Select **More Options** ![More Options](..\images\more-options.png "More Options") > **Format OMF** to reset template content alignment after making changes. 
+
+   - Select **More Options** ![More Options](..\images\more-options.png "More Options") > **Download OMF** to download the message content to your local drive. 
+
+   - Select **More Options** ![More Options](..\images\more-options.png "More Options") > **Reset Template** to delete any changes you have made to the message content. 
+
+   As you edit your message, the editor indicates whether your message is valid in the upper right and displays any errors in the `Errors` pane.
```
Comment (afloresOSI) [inline content/developer-guide/omf-dev/omf-editor.md:29]:
```suggestion
   As you edit a message, the editor indicates whether it message is valid in the upper right and displays any errors in the `Errors` pane.
```

**Review by afloresOSI — COMMENTED** [review]
Just a couple of minor edits. Looks good!

**Review by jasnames — APPROVED** [review]

**Pull request merged**